countries_regions.csv

```diff
@@ -220,7 +220,6 @@ United States,Tennessee
 United States,Texas
 United States,Utah
 United States,Vermont
-United States,Virgin Islands
```

Comment:
Region `Virgin Islands` for country `United States` won't be required for Phase 2